src/patterns/security-code/index.md.njk

```diff
@@ -0,0 +1,135 @@
+
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "12345 is your [service name] security code"
+}) }}
+
+Then ask the user to enter this code:
+
+{{ example({group: "patterns", item: "security-code", example: "default", html: true, nunjucks: true, open: false}) }}
+
+Ignore any formatting or punctuation such as spaces, hyphens and dashes.
+
+Give a time limit of 15 minutes for users to enter the code. If this time limit expires, display a ‘code has expired’ error message and send the user a new code.
+
+If the user follows the ‘Not received a text message?’ link, allow them to check which mobile number they entered, and to change it if necessary. This prevents the user becoming stuck if they entered a mobile number incorrectly. Do not allow users to change the number when signing in.
+
+{{ example({group: "patterns", item: "security-code", example: "resend-first-time", html: true, nunjucks: true, open: false}) }}
+
+This should send a new code, with its own 15 minute time limit. The previous code should remain valid within its original time limit.
+
+### When signing in
+
+When a user returns to your service, verify their password first. Once they have entered this correctly, send a text message to their mobile phone with a 5 digit code in this format:
+
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
+{{ govukInsetText({
+  text: "12345 is your [service name] security code"
+}) }}
+
+Ask the user to enter this code. Use the same pattern and time limit as when creating an account.
+
+If they follow the ‘Not received a text message?’ link, show them a page allowing them to resend the code. Do not reveal the mobile number you sent it to.
+
+{{ example({group: "patterns", item: "security-code", example: "resend", html: true, nunjucks: true, open: false}) }}
+
+You should tell users what to do if they no longer have access to the phone used to sign up.
+
+Depending upon your service, they may need to contact a support address, or may be able to ask a colleague to update their phone number.
+
+### Domain-bound codes
+
+You can also follow the [domain-bound codes](https://developer.apple.com/news/?id=z0i801mg) standard. This improves security by associating the code with a specific domain, allowing browsers to safely autofill it. 
+
+Include the domain of the service on a new line, prefixed with an `@`, followed by a `#` symbol and the code, like this:
+
+{{ govukInsetText({
+html: "12345 is your [service name] security code<br><br>@example.com #12345"
+}) }}
+
+### Error messages
+
+Style error messages like this:
+
+{{ example({group: "patterns", item: "security-code", example: "error-incorrect", html: true, nunjucks: true, open: false}) }}
+
+If the user doesn’t enter enough digits:
+<br>Say ‘You haven’t entered enough numbers’.
```

Comment:
Style guide says not to use negative contractions.

```suggestion
<br>Say ‘You have not entered enough numbers’.
```

Reply:
Good spot, there' s a few other instances of this elsewhere too...

Reply:
All credit to @alexbishop1! 😄